src/components/__tests__/FlawForm.spec.ts

```diff
@@ -12,6 +12,7 @@ import { useRouter } from 'vue-router';
 import LabelDiv from '../widgets/LabelDiv.vue';
 import LabelSelect from '../widgets/LabelSelect.vue';
 import LabelInput from '../widgets/LabelInput.vue';
+import LabelCollapsable from '../widgets/LabelCollapsable.vue'
```

Comment:
We need to fix lint issue. 
![Screenshot from 2024-04-04 09-37-49](https://github.com/RedHatProductSecurity/osim/assets/156366006/a7b9fc73-c6ad-474e-a3eb-f1d173d25a0e)